sdk/cosmos/azure-spring-data-cosmos/README.md

```diff
@@ -1,25 +1,356 @@
 #Azure Cosmos DB client library for Java
 
 ## Getting started
+[Azure Cosmos DB](https://docs.microsoft.com/azure/cosmos-db/introduction) is a globally-distributed database service that allows developers to work with data using a variety of standard APIs, such as SQL, MongoDB, Cassandra, Graph, and Table.
+
+**Spring Data Azure Cosmos DB** provides initial Spring Data support for Azure Cosmos DB using the [SQL API](https://docs.microsoft.com/azure/cosmos-db/sql-api-introduction), based on Spring Data framework. Currently it only supports SQL API, the other APIs are in the plan. 
 
 ## TOC
 
+* [Examples](#Examples)
+* [Spring data version support](#spring-data-version-support)
+* [Feature List](#feature-list)
+* [Quick Start](#quick-start)
+* [Query Partitioned Collection](QueryPartitionedCollection.md)
+* [Beta version package](#Beta version package)
+* [Troubleshooting](#Troubleshooting)
+* [Contributing](#Contributing)
+* [Code of Conduct](#code-of-conduct)
+* [Key concepts](#Key concepts)
+* [Next steps](#Next steps)
+
 ## Examples
+Please refer to [sample project here](./samplecode).
 
 ## Spring Data Version Support
+Version mapping between spring boot and spring-data-cosmosdb:
```

Comment:
Do we need to talk about these versions ?
Since now - it is called - `com.azure:azure-spring-data-cosmos`